src/components/HelpScoutBeacon/IconQuestion.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+const IconQuestion = () => {
+  return (
+    <svg
+      width="15"
+      height="26"
+      viewBox="0 0 15 26"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M9.00619 25.2001H5.8689V22.1497H9.00619V25.2001ZM9.55511 15.516L9.28063 15.7448C9.12349 15.8596 9.00615 16.1264 9.00615 16.3553V19.1381H5.86885V16.3553C5.86885 15.173 6.41781 14.0675 7.32016 13.3429L7.59464 13.1141C10.3394 10.9791 11.672 9.87356 11.672 7.96741C11.6766 6.87409 11.2319 5.82427 10.4368 5.05118C9.6417 4.27808 8.56196 3.84573 7.4375 3.85019C5.0063 3.85019 3.20297 5.60356 3.20297 7.96741H0.0656738C0.0656738 3.96428 3.32031 0.79981 7.4375 0.79981C9.39335 0.797505 11.2698 1.55192 12.6528 2.8966C14.0358 4.24129 14.8117 6.06573 14.8093 7.96741C14.8093 11.4368 12.3781 13.3429 9.55511 15.516Z"
+        fill="white"
+      />
+    </svg>
+  )
+}
```

Comment:
I ran it through `svgr` :smile: 

```jsx
const IconQuestion = () => (
  <svg width={15} height={26} fill="none">
    <path
      d="M9.006 25.2H5.87v-3.05h3.137v3.05zm.55-9.684l-.275.229c-.158.115-.275.381-.275.61v2.783H5.87v-2.783c0-1.182.549-2.287 1.451-3.012l.275-.229c2.744-2.135 4.077-3.24 4.077-5.147a4.043 4.043 0 0 0-1.235-2.91
6 4.28 4.28 0 0 0-3-1.2c-2.43 0-4.234 1.753-4.234 4.116H.066C.066 3.964 3.32.8 7.438.8a7.471 7.471 0 0 1 5.215 2.097 7.06 7.06 0 0 1 2.156 5.07c0 3.47-2.43 5.376-5.254 7.549z"
      fill="#fff"
    />
  </svg>
)
```

Reply:
Thank you, how can I do this myself for one file/icon?

Reply:
Hmmm, does not render the full icon:
![image](https://user-images.githubusercontent.com/3072458/57510412-05c0d680-7307-11e9-956e-b3fbad35f2da.png)

Reply:
argh sorry I didn’t check, it might require some tweaking :s 

Does it work with the previous path?

To run it through `svgr`, I just do `svgr file.svg` 😬